roles/desktop-environment/tasks/main.yaml

```diff
@@ -0,0 +1,285 @@
+    - src: "./src/github.com/scrooloose/nerdtree"
+      dest: "~/.vim/bundle/nerdtree"
+    - src: "./src/github.com/ntpeters/vim-better-whitespace"
+      dest: "~/.vim/bundle/vim-better-whitespace"
+    - src: "./src/github.com/pangloss/vim-javascript"
+      dest: "~/.vim/bundle/vim-javascript"
+    - src: "./src/github.com/tpope/vim-fugitive"
+      dest: "~/.vim/bundle/vim-fugitive"
+    - src: "./src/github.com/tomlion/vim-solidity"
+      dest: "~/.vim/bundle/vim-solidity"
+    - src: "./src/github.com/nvie/vim-flake8"
+      dest: "~/.vim/bundle/vim-flake8"
+    - src: "./src/github.com/ekalinin/Dockerfile.vim"
+      dest: "~/.vim/bundle/Dockerfile"
+    - src: "./src/github.com/junegunn/fzf"
+      dest: "~/.vim/bundle/vim-fzf"
+    - src: "./src/github.com/airblade/vim-gitgutter"
+      dest: "~/.vim/bundle/vim-gitgutter"
+
+
+- name: Link .vim/after
+  file:
+    src: "{{playbook_dir}}/home/vim/after"
+    dest: "~/.vim/after"
+    state: link
+  tags:
+    - vim
+
+- name: Link .vim/colors/base16
+  file:
+    src: "{{playbook_dir}}/src/github.com/chriskempson/base16-vim/colors/base16-tomorrow.vim"
+    dest: "~/.vim/colors/base16-tomorrow.vim"
+    state: link
+  tags:
+    - vim
+
+- name: Link .vim/colors
+  file:
+    src: "{{playbook_dir}}/home/vim/colors"
+    dest: "~/.vim/colors"
+    state: link
+  tags:
+    - vim
+
+- name: Link .vimrc
+  file:
+    src: "{{playbook_dir}}/home/vimrc"
+    dest: "~/.vimrc"
+    state: link
+  tags:
+    - vim
+
+- name: Create .config/awesome
+  file:
+    path: "~/.config/awesome"
+    state: directory
+  tags:
+    - awesome
+
+- name: Link .config/awesome/rc.lua
+  file:
+    src: "{{playbook_dir}}/home/config/awesome/rc.lua"
+    dest: "~/.config/awesome/rc.lua"
+    state: link
+  tags:
+    - awesome
+
+- name: Link .config/awesome/themes
+  file:
+    src: "{{playbook_dir}}/home/config/awesome/themes"
+    dest: "~/.config/awesome/themes"
+    state: link
+  tags:
+    - awesome
+
+- name: Clone github.com/NuckChorris/assault
+  git:
+    dest: "./src/github.com/NuckChorris/assault"
+    repo: "https://github.com/NuckChorris/assault.git"
+    version: master
+
+- name: Link .config/awesome/assault.lua
+  file:
+    src: "{{playbook_dir}}/src/github.com/NuckChorris/assault/awesomewm/assault.lua"
+    dest: "~/.config/awesome/assault.lua"
+    state: link
+  tags:
+    - awesome
+
+- name: Clone github.com/NuckChorris/assault
+  git:
+    dest: "./src/github.com/scottgreenup/lain"
+    repo: "https://github.com/scottgreenup/lain.git"
+    version: master
+
+- name: Link .config/awesome/lain
+  file:
+    src: "{{playbook_dir}}/src/github.com/scottgreenup/lain"
+    dest: "~/.config/awesome/lain"
+    state: link
+  tags:
+    - awesome
+
+- name: Clone github.com/NuckChorris/assault
+  git:
+    dest: "./src/github.com/majutsushi/urxvt-font-size"
+    repo: "https://github.com/majutsushi/urxvt-font-size.git"
+    version: master
+
+- name: Link .config/awesome/lain
```

Comment:
And another :P

Reply:
:man_facepalming: :man_facepalming: :man_facepalming: